SampleApp/SampleApp.iOS/AppDelegate.cs

```diff
@@ -28,6 +27,10 @@ public partial class AppDelegate : global::Xamarin.Forms.Platform.iOS.FormsAppli
 
         public override bool FinishedLaunching(UIApplication app, NSDictionary options)
         {
+            global::Xamarin.Forms.Forms.Init();
```

Comment:
When initializing a Forms app on the native side, the very first thing that has to happen is to initialize Forms, then load the application, then call super FinishedLaunching. Otherwise the application window won't exist when the rest of the the FinishedLaunching executes, and anything in this class that touches the window (such as `FailIfSimulator`) will crash.